runtests.py

```diff
@@ -290,8 +290,31 @@ def main(argv):
     if args.build_only:
         sys.exit(0)
     else:
-        __import__(PROJECT_MODULE)
-        test = sys.modules[PROJECT_MODULE].test
+        try:
+            __import__(PROJECT_MODULE)
+            test = sys.modules[PROJECT_MODULE].test
+            version = sys.modules[PROJECT_MODULE].__version__
+            mod_path = sys.modules[PROJECT_MODULE].__file__
+            mod_path = os.path.abspath(os.path.join(os.path.dirname(mod_path)))
+        except ImportError:
+            current_python_path = os.environ.get('PYTHONPATH', None)
+            print("Unable to import {} from: {}".format(PROJECT_MODULE,
+                                                        current_python_path))
+            dst_dir = os.path.join(ROOT_DIR, 'build', 'testenv')
+            from sysconfig import get_path
+            py_path = get_path('platlib')
+            site_dir = os.path.join(dst_dir, get_path_suffix(py_path, 3))
+            print("Trying to import scipy from development installed path at:",
+                  site_dir)
```

Comment:
Maybe only show this if the import fails? Otherwise the path is shown twice, here and in the `Running tests ...` print.

Reply:
> Maybe only show this if the import fails

That will be shown only when normal import would fail. I think it would be better to keep it there to aware the users that we are actually changing the path of scipy import.

Reply:
okay, makes sense